manifests/init.pp

```diff
@@ -66,7 +69,8 @@
   String $postgresql_db_password = extlib::cache_data('pulpcore_cache_data', 'db_password', extlib::random_password(32)),
   String $django_secret_key = extlib::cache_data('pulpcore_cache_data', 'secret_key', extlib::random_password(32)),
   Integer[0] $redis_db = 8,
-  Stdlib::Fqdn $servername = $facts['fqdn']
+  Stdlib::Fqdn $servername = $facts['fqdn'],
+  Optional[String] $django_remote_user = undef,
```

Comment:
Some bikeshedding: does it make sense to name this closer to the actual setting? When you're familiar with Pulp then you may know `REMOTE_USER_ENVIRON_NAME` but `django_remote_user` doesn't explain itself.

Reply:
We talked it over and we agree with you.